test/functional/api/cas/installer.py

```diff
@@ -14,20 +14,8 @@
 
 
 def install_opencas():
-    TestRun.LOGGER.info("Copying Open CAS repository to DUT")
-    TestRun.executor.rsync_to(
-        f"{TestRun.usr.repo_dir}/",
-        f"{TestRun.usr.working_dir}/",
-        exclude_list=["test/functional/results/"],
-        delete=True)
-
-    TestRun.LOGGER.info("Building Open CAS")
-    output = TestRun.executor.run(
-        f"cd {TestRun.usr.working_dir} && "
-        "./configure && "
-        "make -j")
-    if output.exit_code != 0:
-        raise CmdException("Make command executed with nonzero status", output)
+    download_opencas()
```

Comment:
Maybe 'copy_opencas()' will be a better name for this function, considering what it is doing?